src/ctim/schemas/common.cljc

```diff
@@ -350,6 +364,17 @@
   (vals disposition-map)
   :description "String verdict identifiers")
 
+(def-map-type KillChainPhase
```

Comment:
Why are we not actually defining the kill chain phases as vocabulary here?  They would be the same as those defined in https://en.wikipedia.org/wiki/Kill_chain

- Reconnaissance: Intruder selects target, researches it, and attempts to identify vulnerabilities in the target network.
- Weaponization: Intruder creates remote access malware weapon, such as a virus or worm, tailored to one or more vulnerabilities.
- Delivery: Intruder transmits weapon to target (e.g., via e-mail attachments, websites or USB drives)
- Exploitation: Malware weapon's program code triggers, which takes action on target network to exploit vulnerability.
- Installation: Malware weapon installs access point (e.g., "backdoor") usable by intruder.
- Command and Control: Malware enables intruder to have "hands on the keyboard" persistent access to target network.
- Actions on Objective: Intruder takes action to achieve their goals, such as data exfiltration, data destruction, or encryption for ransom.

Reply:
STIX 2.0 does not link any vocabulary to the [Kill Chain Phase](https://docs.google.com/document/d/1dIrh1Lp3KAjEMm8o2VzAmuV0Peu-jt9aAh1IHrjAroM/edit#heading=h.i4tjv75ce50h). I can add the suggested vocabulary in CTIM.

Reply:
The ATT&CK catalog uses ATT&CK tactics for STIX `kill_chain_phases.phase_name`. As explained in the [ATT&CK introduction and overview](https://attack.mitre.org/wiki/Introduction_and_Overview), the ten tactic categories were derived from the later stages (control, maintain, and execute) of a seven-stage Cyber Attack Lifecycle.

![Image of Yaktocat](https://attack.mitre.org/w/img_auth.php/6/6f/MITRE_attack_tactics.png)

Tactics:

- persistence
- privilege-escalation
- defense-evasion 
- credential-access
- discovery
- lateral-movement
- execution
- collection
- exfiltration
- command-and-control

What vocabulary should we use?

Reply:
In the STIX specification, the Kill Chain Phase has two fields:
- kill_chain_name (ex: mitre-attack, lockheed-martin-cyber-kill-chain)
- phase_name (ex: reconnaissance or weaponization if the kill_chain_name is lockheed-martin-cyber-kill-chain)

The phase_name vocabulary depends on the kill_chain_name.

Should we restrict the kill_chain_name to `lockheed-martin-cyber-kill-chain`?